proto/morpheus.proto

```diff
@@ -1,22 +1,43 @@
 syntax = "proto3";
 
+import "google/protobuf/empty.proto";
+
 package morpheus;
 
 option go_package = "src/protogen";
 
+message PredictedName {
+    string name = 1;
+    uint64 productGroup = 2;
+}
+
 message PredictProductNamesRequest {
-    string receipt = 1;
-    string token = 2;
+    string token = 1;
+    repeated string products = 2;
+    repeated uint64 productGroups = 3;
 }
 
 message PredictedProductNames {
-    repeated string predictedName = 1;
+    repeated PredictedName predictedName = 1;
```

Comment:
```suggestion
    repeated PredictedName predictedNames = 1;
```

Reply:
I'll commit that seperately, simultaneously with generated code